Sources/CouchDBSample/main.swift

```diff
@@ -91,128 +81,126 @@ let connProperties = ConnectionProperties(
     host: host,         // httpd address
     port: port,         // httpd port
     secured: secured,   // https or http
-    username: username, // username
-    password: password  // password
+    username: nil,      // admin username
+    password: nil       // admin password
 )
 
-Log.info("Connection Properties:\n\(connProperties)")
+print("Connection Properties:\n\(connProperties)")
 
 // Create couchDBClient instance using conn properties
```

Comment:
conn properties should be either connProperties or probably better "connection properties" in this comment.